content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -5,86 +5,172 @@ weight: 2
 
 OpenTelemetry for .NET is unique among OpenTelemetry implementations, as it is integrated with the .NET `System.Diagnostics` library. At a high level, you can think of OpenTelemetry for .NET as a bridge between the telemetry available through `System.Diagnostics` and the greater OpenTelemetry ecosystem, such as OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector. 
 
-# Installation
+## Installation
 
-You can find OpenTelemetry packages on [NuGet](https://www.nuget.org/profiles/OpenTelemetry). Install them to your project file using the `dotnet` command line utility or through `PackageReference` statements in your `csproj` file.
+OpenTelemetry is available as a [NuGet package](https://www.nuget.org/packages/OpenTelemetry/). Install it with your preferred package manager client.
 
-# Initialization and Configuration
+For example, using the .NET CLI:
 
-OpenTelemetry should be configured as part of your services initialization. In ASP.NET Core, you'll want to add it to the `IServiceCollection` that is created in `public void ConfigureServices(IServiceCollection services)`, in `Startup.cs`. For ASP.NET, configuration occurs in `Global.asax.cs`.
+```console
+dotnet add package OpenTelemetry
```

Comment:
The `console` style only makes sense if we include a prompt in front of commands. Would the standard `$` prompt make sense here, if not, what would you suggested? (This comment applies to other `console` code blocks below.)